docs/r2/tensorboard_profiling_keras.ipynb

```diff
@@ -0,0 +1,1086 @@
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "W0425 21:14:51.180765 140078590396288 callbacks.py:238] Method (on_train_batch_end) is slow compared to the batch update (0.134130). Check your callbacks.\n"
+          ],
+          "name": "stderr"
+        },
+        {
+          "output_type": "stream",
+          "text": [
+            "20/20 [==============================] - 51s 3s/step - loss: 5.3766 - categorical_accuracy: 0.1004\n",
+            "Epoch 2/5\n",
+            "20/20 [==============================] - 5s 227ms/step - loss: 4.8007 - categorical_accuracy: 0.0988\n",
+            "Epoch 3/5\n",
+            "20/20 [==============================] - 5s 242ms/step - loss: 4.3439 - categorical_accuracy: 0.0980\n",
+            "Epoch 4/5\n",
+            "20/20 [==============================] - 5s 247ms/step - loss: 3.9405 - categorical_accuracy: 0.1074\n",
+            "Epoch 5/5\n",
+            "20/20 [==============================] - 5s 225ms/step - loss: 3.6195 - categorical_accuracy: 0.1176\n"
+          ],
+          "name": "stdout"
+        },
+        {
+          "output_type": "execute_result",
+          "data": {
+            "text/plain": [
+              "<tensorflow.python.keras.callbacks.History at 0x7f65f8758908>"
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 12
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "042k7GMERVkx",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Visualizing profile result using TensorBoard\n",
+        "\n",
+        "Unfortunately, due to [#1913](https://github.com/tensorflow/tensorboard/issues/1913), you cannot use TensorBoard in Colab to visualize profile result. You are going to download the logdir and start TensorBoard on your local machine.\n",
+        "\n",
+        "Compress logdir:\n"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "6pck56gKReON",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "!tar -zcvf logs.tar.gz logs/profile/"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "TZOf_K4L-Nkv",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Download **logdir.tar.gz** by right-clicking it in **Files** tab.\n",
+        "\n",
+        "![Download](https://github.com/tensorflow/tensorboard/blob/master/docs/r2/images/profiler-download-logdir.png?raw=1\\)\n",
+        "\n",
+        "Please make sure you have the latest [TensorBoard](https://www.tensorflow.org/tensorboard) installed on you local machine as well. Execute following commands on your local machine:\n",
```

Comment:
I installed the latest tb-nightly and tried to visualize the logs created by the tutorial. The profile dashboard gets stuck on "Processing datasets" with the warnings (in case they are relevant):
```
W0429 15:25:10.113260 123145552482304 core_plugin.py:172] Unable to get first event timestamp for run profile/20190429-222000/train: No event timestamp could be found
W0429 15:25:10.174510 123145552482304 core_plugin.py:172] Unable to get first event timestamp for run profile/20190429-222000/train: No event timestamp could be found
```

Reply:
Did you see SCALAR? Did you call pip with "--force-reinstall" flag?

Reply:
That seemed to work!